requirements.txt

```diff
@@ -1,5 +1,5 @@
 numpy
 progress
-python>=3.6
-tensorflow>=2.0
```

Comment:
we need tf2

Reply:
Hm okay, problem was fixing the Python version
https://github.com/ComPWA/tensorwaves/pull/22/commits/a2652e664ee226c0eafafc24fda5b65eb50c4283